repos/exporter/lib/exporter-stack.ts

```diff
@@ -121,43 +152,50 @@ export class ExporterStack extends cdk.Stack {
             logRetention: 180,
         });
 
-        this.addAlarmsToLambda(stage, bastionTerminatorFunction, `bastion-terminator-${stage}`, 30000);
+        this.addAlarmsToLambda(bastionTerminatorFunction, `bastion-terminator-${this.stage}`, 30000);
 
         bastionTerminatorFunction.addToRolePolicy(
             new PolicyStatement({ actions: ['ec2:TerminateInstances', 'ec2:DescribeInstances'], resources: ['*'] }),
         );
 
-        new Rule(this, `bastion-terminator-rule-${stage}`, {
+        new Rule(this, `bastion-terminator-rule-${this.stage}`, {
             schedule: Schedule.cron({ weekDay: 'MON', minute: '0', hour: '6' }), // every monday at 6am
             targets: [new LambdaFunction(bastionTerminatorFunction)],
         });
+    }
 
-        const netexBucket = Bucket.fromBucketName(this, 'netex-bucket', `fdbt-netex-data-${stage}`);
-        const zipperFunction = new NodejsFunction(this, `zipper-${stage}`, {
-            functionName: `zipper-${stage}`,
-            entry: './lib/zipperHandler.ts',
+    private addReferenceDataLambdas(securityGroup: ISecurityGroup, vpc: IVpc, vpcSubnets: { subnets: ISubnet[] }) {
+        const tableRenameFunction = new NodejsFunction(this, `table-rename-${this.stage}`, {
+            functionName: `table-rename-${this.stage}`,
+            entry: './lib/referenceData/tableRenameHandler.ts',
             environment: {
-                NETEX_BUCKET: netexBucket.bucketName,
-                MATCHING_DATA_BUCKET: matchingDataBucket.bucketName,
-                STAGE: stage,
+                RDS_HOST: Fn.importValue(`${this.stage}:RdsClusterInternalEndpoint`),
+                STAGE: this.stage,
             },
+            securityGroups: [securityGroup],
+            vpc: vpc,
+            vpcSubnets: vpcSubnets,
             bundling: {
                 sourceMap: true,
                 sourceMapMode: SourceMapMode.DEFAULT,
             },
             timeout: Duration.minutes(15),
             logRetention: 180,
-            memorySize: 2048,
         });
 
-        netexBucket.grantReadWrite(zipperFunction);
+        this.addAlarmsToLambda(tableRenameFunction, `tableRenameFunction-${this.stage}`, 420000);
+
+        tableRenameFunction.addToRolePolicy(new PolicyStatement({ actions: ['ssm:GetParameter'], resources: ['*'] }));
```

Comment:
Can the resources be restricted here and explicitly list only the parameters it needs, rather than `*`?

Same for the other two Lambdas, but appreciate they're not being added by this PR

Reply:
When I paired with Rob we wrote some code to do this, but it seems to have been lost

Reply:
Didn't see that code, I'll see if I can find it